### PR TITLE
Adds isList function

### PR DESCRIPTION
**Pull request by cperryk**
Needed for [Trello](https://trello.com/c/Z0tqynOw/36-add-site-to-site-import-in-clay-cli)

### CODE REVIEW

File: lib/isList/index.test.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    expect(nonStringArgument).to.throw(Error);
+  });
+
+  it('returns false if non-list reference', () => {
+    expect(fn('domain.com/users/foo')).to.equal(false);
+    expect(fn('domain.com/pages/foo')).to.equal(false);
+    expect(fn('domain.com/components/foo')).to.equal(false);
+    expect(fn('domain.com/components/foo/instances/bar')).to.equal(false);
+  });
+});
```
Comment (reubenson) [inline lib/isList/index.test.js:26]: do we also want to test that fn('domain.com/lists/') is false?

File: lib/isList/index.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+/**
+ * First test if argument is a String. If true, test if '/lists/' is in the string.
+ * Otherwise, throw an error.
+ * @param  {string}  uri
+ * @return {Boolean}
+ */
+module.exports = function (uri) {
+  isUriStringCheck.strCheck(uri);
+
+  return uri.toLowerCase().indexOf('/lists/') > -1;
```
Comment (amycheng) [inline lib/isList/index.js:14]: could we use `includes()` instead? it would be more concise

**Review by amycheng — APPROVED** [review]
lgtm, just a minor comment

File: lib/isList/index.test.js (hunk 1 above)
Comment (cperryk) [reply lib/isList/index.test.js:26]: Good point. That would be smarter but inconsistent with the behavior of the other fncs, so we should probably cut a separate PR that fixes it across all. Will create an issue.

Comment (reubenson) [reply lib/isList/index.test.js:26]: sounds good

**Review by reubenson — APPROVED** [review]

**Pull request merged**